config/nav.yml

```diff
@@ -202,15 +222,6 @@ nav:
           - Available Channels: eventing/channels/channels-crds.md
           - Subscriptions: eventing/channels/subscriptions.md
         - Sugar Controller: eventing/sugar/README.md
```

Comment:
-> Sugar controller

Reply:
I'll leave this alone for now please since it's outside the scope of what I'm trying to do with this PR 🙂 don't want to add too much scope creep